docker/pypi/wmagent-mariadb/Dockerfile

```diff
@@ -0,0 +1,81 @@
+ARG TAG=11.0
+ARG MDB_TAG=$TAG
+FROM mariadb:${MDB_TAG}
+MAINTAINER Valentin Kuznetsov vkuznet@gmail.com
+
+ARG MDB_TAG=$TAG
+ENV MDB_TAG=${MDB_TAG}
+RUN echo MDB_TAG=$MDB_TAG
+
+RUN apt-get update && apt-get install -y vim less sudo wget unzip python3 pip
+
+# Install some debugging tools
+RUN apt-get install -y hostname net-tools iputils-ping procps emacs-nox tcpdump && apt-get clean
+
+ENV USER=cmst1
+# ENV MDB_PORT=
+ENV UID=31961
+ENV MDB_ROOT_DIR=/data
+
+ENV MDB_BASE_DIR=$MDB_ROOT_DIR/srv/mariadb
+ENV MDB_ADMIN_DIR=$MDB_ROOT_DIR/admin/mariadb
+ENV WMA_ADMIN_DIR=$MDB_ROOT_DIR/admin/wmagent
```

Comment:
unless `WMA_ADMIN_DIR` is explicitly required in a code you may collapse `MDB_ARMIN_DIR` and `WMA_ADMIN_DIR` and assign them to `$MDB_ROOT_DIR/admin`, then use only one env variable below.

Reply:
Well, The only difference is for the way how we treat passwords and which one is preserved  where. So far we I have configured two passwords files: `WMAgent.secrets` and `MariaDB.secrets`. 
* The later is holding the MariaDB root account password and is relevant to the container setup/configuration (but still preserved with the system database on the host mount point) . This account has full administrative power over the Database server. I used the user  which is running the service for the  purpose -  `cmst1` 
* The former one is holding the  WMAgent  user only, which is the one  to be used from the WMAgent to connect to the database. This user has access only to the `wmagent` database. And is related to the WMAgent configuration which is supposed to be running on the same host, but not at the same container.

So that was  my argumentation to separate those tow accounts configurations in separate files. If you think it is too precautious   I can merge them back in one and he same file  and use only the WMAgent.secrets file. But this means every WMAgent service running at the same host (not yet the case, though) must also have the database root account configuration as well.